pkg/util/util.go

```diff
@@ -122,3 +124,28 @@ func GenFileUrl(location string, os string) string {
 	}
 	return "file://" + urlPath
 }
+
+// Convert String Array of Parameters to a Map[String]string
```

Comment:
Please add the expected format of the parameter to the function comments
And it would be nice to have the name of the function to indicate that it converts slice of Key-Value strings to map